woocommerce/woocommerce-android
# Update UI to display analytics configuration

Closes: #10703

### Why
As part of Expanded Analytics Hub milestone 2, we plan to add support for customizing the cards we display on the Analytic Hub screen.

### Description
This PR updates the way cards are displayed on the Analytic Hub screen. With the changes introduced here, the Hub screen will have a recycler view of analytics cards, making it easy to display/hide cards and reorder them based on the selected analytics configuration by the merchant.

### Testing instructions
TC1
1. Open the app
2. Tap on See more
3. Check that the Analytic Hub screen loads as expected
4. Tap on the pencil at the top of the screen
5. Edit the Analytics configuration (reorder, hide, show cards)
6. Save the configuration
7. Check that the cards are displayed based on the saved configuration (repeat steps 5-7 with different configurations)

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/0f4d8ca0-269f-461f-95bd-45d07aedb28f
